crates/messaging/Cargo.toml

```diff
@@ -22,6 +22,8 @@ crossbeam-channel = "0.5"
 url = "2"
 async-trait = { workspace = true }
 tokio = { version = "1", features = ["full"] }
+# messaging.filesystem deps
+filesystem-pubsub = { git = "https://github.com/danbugs/filesystem-pubsub", rev = "45f369ea58d98a8b1d273a9a1fddbceb773fc607" }
```

Comment:
I would publish this as a crate to crates.io instead of linking to a github URL. This can come later.

Reply:
For sure, I'll publish it!